presqt/api_v1/utilities/io/zip_file.py

```diff
@@ -2,19 +2,23 @@
 import zipfile
 
 
-def zip_directory(destination_path, source_path):
+def zip_directory(destination_path, source_path, to_strip=None):
```

Comment:
Total nitpick..
Most tools that I've seen usually put source first, then destination.

For example... linux copy/move commands.